0136-Single-Number.py

```diff
@@ -0,0 +1,26 @@
+'''
+Given a non-empty array of integers, every element appears twice except for one. Find that single one.
+
+Note:
+
+Your algorithm should have a linear runtime complexity. Could you implement it without using extra memory?
+
+Example 1:
+
+Input: [2,2,1]
+Output: 1
+Example 2:
+
+Input: [4,1,2,1,2]
+Output: 4
+'''
+class Solution:
+    def singleNumber(self, nums: List[int]) -> int:
+        if len(nums) == 1:
+               return nums[0]
+        while True:
+            a = nums.pop(0)
+            if a in nums:
```

Comment:
This is not linear time because for every element you are checking if that element is in the array or not which means you are traversing array for every a.